src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,49 +1,83 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  closeTodo: () => void,
+  selectedTodo: Todo | null,
+};
+
+export const TodoModal: React.FC<Props> = ({ selectedTodo, closeTodo }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isError, setIsError] = useState(false);
+
+  const fetchData = async () => {
+    try {
+      const currentUser = await getUser(selectedTodo?.userId || 0);
+
+      setUser(currentUser);
+    } catch {
+      setIsError(true);
+    }
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
-        <Loader />
-      ) : (
-        <div className="modal-card">
-          <header className="modal-card-head">
-            <div
-              className="modal-card-title has-text-weight-medium"
-              data-cy="modal-header"
-            >
-              Todo #2
-            </div>
+      {!user
+        ? (
+          <Loader />
+        )
+        : (
```

Comment:
```suggestion
      {!user ? (
        <Loader />
      ) : (
```
looks better if format it in this way (just FYI)